server/auth/authrequest.js

```diff
@@ -54,9 +54,12 @@ function performRequest(endpoint, method, data, fiwareHeaders) {
       resolveWithFullResponse: true,
       json: true // Automatically stringifies the body to JSON
     };
+    if (typeof payload !== 'undefined') {
+      endpoint += '?' + querystring.stringify(payload);
+      options.uri = host + ':' + port + endpoint;
```

Comment:
I've added this to allow passing the `keyValues` option when doing POST or PATCH. Before the model change and Orion update, the client was using this format to PATCH the elements so, we need to use `keyValues` to not break the compatibility